ext/standard/array.c

```diff
@@ -6581,6 +6581,165 @@ PHP_FUNCTION(array_filter)
 }
 /* }}} */
 
+/* {{{ Internal function to find an array element for a user closure. */
+static zend_result php_array_find(const HashTable *array, zend_fcall_info fci, zend_fcall_info_cache fci_cache, zval *result_key, zval *result_value, bool negate_condition)
```

Comment:
Anyone have an objection to making this `ZEND_API`?

Reply:
I have no disagreement. I have chosen a lower visibility for the time being, as it is always easier to increase visibility afterwards instead of reducing it. But if you see a more global use case for the method, we can make it `ZEND_API`.

Reply:
I've now merged this as-is. The `PHPAPI` can be added in a follow-up commit if necessary/desired.